letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if (integer >= 90)
+    return "A"
+  elsif (integer >= 80)
+    return "B"
+  elsif (integer >= 70)
+    return "C"
+  elsif (integer >= 60)
+    return "D"
+  else
+    return "F"
+  end
```

Comment:
You don't need `;`, you can use the `then` keyword instead for one line if/elsif/else conditions.